### PR TITLE
UIIN-1103 Show two instance records as a split screen - ViewInstance refactoring

### PR DESCRIPTION
**Pull request by NikitaSedyx**
## Purpose

https://issues.folio.org/browse/UIIN-1103
to avoid copy-paste in instances split screen implementation

## Approach

move views in small separate components

#### TODOS and Open Questions

more refactoring

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/Instance/InstanceDetails/InstanceNotesView/InstanceNotesView.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+import InstanceNotesList from './InstanceNotesList';
+
+const InstanceNotesView = ({
+  id,
+  instance,
+  noteTypes,
+}) => {
+  const notesGroups = useMemo(() => {
+    if (!instance.notes?.length) {
```
Comment (VictorSoroka1) [inline src/Instance/InstanceDetails/InstanceNotesView/InstanceNotesView.js:22]: `isEmpty` from lodash
```suggestion
    if (isEmpty(instance.notes)) {
```

File: src/Instance/InstanceDetails/InstanceTitleData/AlternativeTitlesList.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+import {
+  checkIfArrayIsEmpty,
+} from '../../../utils';
+
+const noValue = <NoValue />;
+
+const rowMetadata = ['alternativeTitleTypeId'];
+const visibleColumns = ['type', 'title'];
+const getColumnMapping = intl => ({
+  type: intl.formatMessage({ id: 'ui-inventory.alternativeTitleType' }),
```
Comment (VictorSoroka1) [inline src/Instance/InstanceDetails/InstanceTitleData/AlternativeTitlesList.js:21]: Respect for not using translations as keys which were the case in the initial implementation.

File: src/ViewInstance.js [hunk 3]
```diff
@@ -641,7 +570,7 @@ class ViewInstance extends React.Component {
       acc03: !areAllFieldsEmpty([identifiers]),
       acc04: !areAllFieldsEmpty([contributors]),
       acc05: !areAllFieldsEmpty(values(descriptiveData)),
-      acc06: !areAllFieldsEmpty([instanceNotes]),
+      acc06: !areAllFieldsEmpty([[get(instance, 'notes', [])]]),
```
Comment (mkuklis) [inline src/ViewInstance.js:573]: @NikitaSedyx could we use `instance?.nodes ?? []` instead?

Comment (NikitaSedyx) [reply src/ViewInstance.js:573]: yep, will change

File: src/ViewInstance.js [hunk 4]
```diff
@@ -641,7 +570,7 @@ class ViewInstance extends React.Component {
       acc03: !areAllFieldsEmpty([identifiers]),
       acc04: !areAllFieldsEmpty([contributors]),
       acc05: !areAllFieldsEmpty(values(descriptiveData)),
-      acc06: !areAllFieldsEmpty([instanceNotes]),
+      acc06: !areAllFieldsEmpty([instance?.nodes ?? []]),
```
Comment (NikitaSedyx) [inline src/ViewInstance.js:573]:
```suggestion
      acc06: !areAllFieldsEmpty([instance?.notes ?? []]),
```

**Pull request merged**